src/screens/InputUsername/index.js

```diff
@@ -284,8 +285,8 @@ const ChooseUsername = () => {
                         : DEFAULT_PROFILE_PIC_PATH,
                       cache: 'reload'
                     }}
-                    width={52}
-                    height={52}
+                    width={dimen.normalizeDimen(52)}
+                    height={dimen.normalizeDimen(52)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving the responsiveness of the application. By using `dimen.normalizeDimen`, the dimensions of the image will be adjusted based on the screen size, which can help to ensure that the layout looks consistent across different devices.



<!-- This is an auto-generated comment by OSS CodeRabbit -->